test/iron-image.html

```diff
@@ -207,6 +207,33 @@
 
             image.src = randomImageUrl();
           });
+
+          // Tests for PolymerElements/iron-image#115
+          test('image is not loaded when prevent-load is set', function() {
+            var attemptedLoadSpy = sinon.spy(image, '_setLoading');
+            image.preventLoad = true;
+            image.src = randomImageUrl();
+
+            expect(attemptedLoadSpy).not.to.have.been.calledWith(true);
+            expect(image._resolvedSrc).not.to.be.ok;
+          });
+          test('image is loaded when prevent-load has been removed after src changed', function(done) {
```

Comment:
Okay :+1: Was grouping the tests under that comment. Will add space though based on previous comment discussion.